src/index.css

```diff
@@ -1,13 +1,28 @@
-body {
+* {
```

Comment:
Just a reminder to be carefull on applying styles to all the elements in the doc :), we should not populate it too much for best practices